.github/workflows/eslint-ci.yml

```diff
@@ -19,4 +19,4 @@ jobs:
       - name: Install dependencies and run eslint
         run: |
           pnpm install
-          pnpm eslint
+          pnpm turbo run lint
```

Comment:
Why not do `pnpm lint` here, since it does that below?

```
 "lint": "pnpm turbo run lint",
```

Reply:
I tried both ways and GH Actions doesn't run the command at all. I need to look more into it.